Doc/library/importlib.rst

```diff
@@ -124,6 +124,11 @@ Functions
     need to call :func:`invalidate_caches` in order for the new module to be
     noticed by the import system.
 
+    If the module cannot be imported, :func:`import_module` will raise an
+    :exc:`ImportError`. If the module is found and loaded, but the code in the
+    module raises an exception, :func:`import_module` will pass that exception
+    to the module that called it.
```

Comment:
```suggestion
    If the module cannot be imported, :func:`import_module` will raise
    :exc:`ImportError` or an appropriate subclass like
    :exc:`ModuleNotFoundError`.
```



I don't think there's any benefit to saying exceptions will propagate as that's the default for any Python code.

I also think we should make sure that people are aware that subclasses may be raised as well since typically people really want `ModuleNotFoundError`.

Reply:
I suggest to look further down, where I tried to consolidate things. (sorry for the spam, I hope it's helpful for readers 😅)